stories/topics/proc-aap-aws-extension-nodes.adoc

```diff
@@ -0,0 +1,198 @@
+[id="proc-aap-aws-extension-nodes"]
+
+Tech Preview
```

Comment:
For Tech Preview, see the corresponding GCP documentation for how to mark something as a tech preview.